hudi-integ-test/pom.xml

```diff
@@ -100,6 +98,21 @@
       <scope>test</scope>
     </dependency>
 
+    <!-- Parquet -->
+    <dependency>
+      <groupId>org.apache.parquet</groupId>
+      <artifactId>parquet-avro</artifactId>
+      <version>${parquet.version}</version>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.parquet</groupId>
+      <artifactId>parquet-hadoop</artifactId>
+      <version>${parquet.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
why are these dependencies needed here? also, if we run integ tests with hudi-spark3.x-bundle, wouldn't they already be present in classpath?

Reply:
maybe parquet-hadoop is not required. i'll have to re-run to confirm

Reply:
> this actually affects Flink's IT. consider this is only test deps, did not dig further

scratch this. this is caused by a newly added testcase that only runs with flink 1.17